charts/victoria-metrics-distributed/_index.md.gotmpl

```diff
@@ -76,9 +76,9 @@ To keep data completeness for all the availability zones, make sure you have eno
 
 And to avoid getting incomplete responses from `zone-eu-1` which gets recovered from outage, check vmagent on `zone-us-1` to see if persistent queue has been drained. If not, remove `zone-eu-1` from serving query by setting `.Values.availabilityZones.{zone-eu-1}.read.allow=false` and change it back after confirm all data are restored.
 
-### How to use [multitenancy](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#multitenancy)?
+### How to use multitenancy?
```

Comment:
This potentially could be a breaking change, but I did since the anchor with the link is not clickable from the side menu.